test/blackbox-tests/test-cases/too-many-parens/run.t

```diff
@@ -25,8 +25,19 @@ are readable.
   These parentheses are no longer necessary with dune, please remove them.
   [1]
 
-Checking that extra long stanzas (over 10 lines) are not printed
+Checking that extra long stanzas (over 10 lines) are truncated in the middle, and the two blocks are aligned.
   $ dune build --root d
   File "dune", line 3, characters 13-192:
+   3 |   (libraries (a
```

Comment:
I notice that the truncated view always displays 11 lines. This means that 10 lines will be displayed in full, but 11 lines will be displayed with 3 lines missing, while keeping the same number of lines. It's a bit unfortunate. I can see three things that we can do:
- increase the threshold (display more messages in full)
- remove some context lines (e.g. switch to 3)
- remove the empty lines

I think that 3 context lines would be enough, but that's up to you.